istioctl/pkg/version/version.go

```diff
@@ -214,6 +215,17 @@ func xdsProxyVersionWrapper(xdsResponse **discovery.DiscoveryResponse) func() (*
 					ID:           node.Id,
 					IstioVersion: getIstioVersionFromXdsMetadata(node.Metadata),
 				})
+			case "type.googleapis.com/envoy.service.status.v3.ClientConfig":
```

Comment:
I think this actually also fixed a bug where the ztunnel proxy was not counted in the `x version` :)

https://github.com/istio/istio/blob/1465f9f212975059554e5ba56bbfd833d6d6c60d/istioctl/pkg/version/version.go#L210-L213